.github/workflows/main.yml

```diff
@@ -33,8 +34,20 @@ jobs:
         run: |
           nix-instantiate --json --eval --strict -E 'with builtins; attrNames (getFlake (toString ./.)).checks.${currentSystem}' | perl -pe 's|(.*)|checks=\1|' >>$GITHUB_OUTPUT
 
+      - name: Generate Version 🏷️
+        id: version
+        run: |
+          TAG_COUNT=$(git rev-list --tags --no-walk --count)                                                                 # Count all tags
+          COMMIT_COUNT=$(git rev-list --use-bitmap-index --count $(git rev-list --tags --no-walk --max-count=1)..HEAD)       # Count all commits since the last tag
```

Comment:
Same with this, also since we probably bump flake.nix a lot this number is rather infalted.

Reply:
Everyone using this is running at least one Microsoft product and should therefore be accustomed to ludicrously high version numbers like `10.0.22623.1245`

Jokes aside: I don't think that the commit count will ever exceed three (or at the very most four) digits and that's totally acceptible